From: Maverick <mavjs01@gmail.com>
Date: Sat, 14 Nov 2020 13:00:47 +0100
Subject: Fix missing -T option in getopts declaration.

* The -T option to define a file-of-targets is mentioned in the pdf as
  well as the usage section, however, it failed to include it as part of
  the getopts declaration in line:87
---
 CHANGELOG         | 3 +++
 smtp-user-enum.pl | 2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/CHANGELOG b/CHANGELOG
index 4c23d06..eb0b77d 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,6 @@
+2020-08-17 smtp-user-enum v1.3
+
+* Bug fix: -T option works now.
 
 2008-04-20 smtp-user-enum v1.2
 
diff --git a/smtp-user-enum.pl b/smtp-user-enum.pl
index 8735270..c6f073e 100755
--- a/smtp-user-enum.pl
+++ b/smtp-user-enum.pl
@@ -84,7 +84,7 @@ Examples:
 
 USAGE
 
-getopts('m:u:U:s:S:r:dt:vhM:f:D:p:w:', \%opts);
+getopts('m:u:U:s:S:r:dt:T:vhM:f:D:p:w:', \%opts);
 
 # Print help message if required
 if ($opts{'h'}) {
